boxd00-sp

boxandav opened the pull request:
Základní struktura aplikace
Omlouvám se za pozdní odevzdání, zpočátku nefungovalo kvůli Commit erroru, jak jsem zmínil v issue

boxandav committed:
pilotní verze

nvbach91 commented:
prace neni odevzdana

boxandav committed:
finální verze

bez složek vendor a storage, omlouvám se za pozdní odevzdání

nvbach91 commented:
- nova stranka pro sedadla konkretniho letu, nezobrazovat sedadla v prehledu letu
- strankovani letu
- administrace letenek
- aplikac eneni dostupna na internetu
- pozdni odevzdani
- neni oauth

boxandav committed:
upravená finální verze

hotová správa destinací, spojení a letů
nahráno na internet, odkaz https://eso.vse.cz/~boxd00/app

bohužel jsem nestihl všechny úkoly, chtěl bych se zeptat, bylo by je možné dodělat případně do dneška?

boxandav committed:
upravená finální verze + SQL

boxandav commented:
upravená finální verze odevzdána i s SQL
odkaz: https://eso.vse.cz/~boxd00/app/
